_data/profiles/remy-bernard.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: Rémy Bernard
+role:
+  en: Genius
```

Comment:
I have some doubts about the validity of the supposed facts listed on this profile's data... :thinking:

Reply:
I will support it by an Expert opinion:

It is true

😄

Reply:
Wait, you'll bring in an impartial 3rd party?